main.go

```diff
@@ -1,21 +1,102 @@
-// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 package main
 
-import "github.com/giantswarm/leanix-exporter/cmd"
+import (
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/giantswarm/leanix-exporter/flag"
```

Comment:
We tend to split our imports up a little:

- standard lib (math/rand, eg)
- third party (viper)
- internal (giantswarm/leanix-exporter)

no biggie, just an fyi